cohort-engine/src/main/java/com/ibm/cohort/engine/measure/seed/MeasureEvaluationSeeder.java

```diff
@@ -98,27 +98,31 @@ public IMeasureEvaluationSeed create(Measure measure, String periodStart, String
 				.map(ParameterDefinition.class::cast)
 				.forEach(parameterDefinition -> setDefaultValue(context, parameterDefinition, dataProvider));
 
+		if (parameters != null) {
+			parameters.entrySet().stream()
+					.forEach(e -> context.setParameter(null, e.getKey(), e.getValue().toCqlType()));
+
+		}
+
+		// Set measurement perdiod last to make sure we respect perdiodStart
+		// and periodEnd date boundaries for an execution.
+		Interval measurementPeriod = createMeasurePeriod(periodStart, periodEnd);
+		context.setParameter(null, MEASUREMENT_PERIOD, measurementPeriod);
+
 		return new CustomMeasureEvaluationSeed(measure, context, measurementPeriod, dataProvider);
 	}
 
 	protected Context createContext(
 			org.cqframework.cql.elm.execution.Library library,
 			String modelUri,
 			DataProvider dataProvider,
-			Interval measurementPeriod,
 			String productLine) {
 
 		Context context = createDefaultContext(library);
 		context.registerLibraryLoader(libraryLoader);
 		context.registerTerminologyProvider(terminologyProvider);
 		context.registerDataProvider(modelUri, dataProvider);
 
-		context.setParameter(
-		null,
-                "Measurement Period",
```

Comment:
Ok. Clearly hard-coded prior to your work. We could deal with this issue of the blessed parameter name in a separate ticket as needed.

Reply:
I think this was copied over from cqf-ruler: https://github.com/DBCG/cqf-ruler/blob/29e5a37fad5ec1890d74c0b087920049e76ef30d/r4/src/main/java/org/opencds/cqf/r4/evaluation/MeasureEvaluationSeed.java#L97